prowler/providers/aws/services/ssm/ssm_service.py

```diff
@@ -145,6 +146,7 @@ def __describe_instance_information__(self, regional_client):
                         id=resource_id,
                         region=regional_client.region,
                     )
+                time.sleep(0.1)
```

Comment:
The boto3 paginator should be handling the throttling exceptions and retrying, but it doesn't work consistently for ssm:DescribeInstanceInformation and exhausts retries.  We escalated this to AWS support and their recommendation was "Reduce the frequency of the API calls", which is why I've been using a short sleep between pages like this which has been working.  The rate limits for this action are not documented and seem to behave differently from other AWS actions, and the problem is difficult to test for because it only shows up when you have large numbers of instances running, in our case ephemeral Spot processing jobs.  When the problem occurs it generates large numbers of false positives because the ssm instance metadata is not captured and ec2_instance_managed_by_ssm fails for most of the instances, and we have not seen that happen after adding the sleep.

Reply:
Could you add this message as a comment above the time sleep? It'd be great to have this.

Thanks for your analysis 👏

Reply:
Good idea, done.